lib/commands/actions.js

```diff
@@ -312,6 +312,20 @@ commands.networkSpeed = async function networkSpeed (networkSpeed) {
   await this.adb.networkSpeed(networkSpeed);
 };
 
+/**
+ * Emulate sensors values on the connected emulator.
+ *
+ * @param {string} sensorType - Sensor type declared in adb.SENSORS items.
+ * @param {number|string} value  - Number to set as the sensor value.
+ * @throws {Error} - If deviceType is not an emulator
+ */
+commands.sensorSet = async function sensorSet (sensorType, value) {
```

Comment:
which external API endpoint is supposed to trigger this method?

Reply:
@mykola-mokhnach this is something I want to discuss with the community. We have two options here; 
1. using one endpoint per type of sensor. For ex; W3C already has a method for light sensor in progress. https://www.w3.org/TR/ambient-light/
2. using one endpoint for all sensors. 
I'd like to hear you guys feedback. I'll probably start the conversation later today in slack

Reply:
I would say it is always safe to have a "mobile:" endpoint for such purpose

Reply:
Added a new mapping here: https://github.com/appium/appium-android-driver/pull/555/files#diff-2ac213fe13b12323b4767dd0bd037ee4R37